src/osip-simulation-view-interface/src/main/java/edu/kit/pse/osip/simulation/controller/SimulationControlInterface.java

```diff
@@ -37,4 +37,8 @@
      * Show the control window
      */
     void show();
+    /**
+     * Closes the window
+     */
+    void close();
```

Comment:
Achso ok. Ich dachte die JavaFX API kennt auch nur hide(). Dann ist es in Ordnung